--- a/include/iconv.h.in
+++ b/include/iconv.h.in
@@ -25,7 +25,7 @@ extern "C" {
 #endif
 
 #define _LIBICONV_VERSION 0x0112    /* version number: (major<<8) + minor */
-extern @DLL_VARIABLE@ int _libiconv_version; /* Likewise */
+#define _libiconv_version _LIBICONV_VERSION /* Likewise */
 
 #ifdef __cplusplus
 }
--- a/lib/iconv.c
+++ b/lib/iconv.c
@@ -660,7 +660,6 @@ const char * iconv_canonicalize (const char * name)
   return name;
 }
 
-int _libiconv_version = _LIBICONV_VERSION;
 
 #if defined __FreeBSD__ && !defined __gnu_freebsd__
 /* GNU libiconv is the native FreeBSD iconv implementation since 2002.
