--- a/pkgs/itcl4.3.4/win/dllEntryPoint.c
+++ b/pkgs/itcl4.3.4/win/dllEntryPoint.c
@@ -41,10 +41,7 @@
 #ifndef STATIC_BUILD
 
 BOOL APIENTRY
-DllEntryPoint(hInst, reason, reserved)
-    HINSTANCE hInst;		/* Library instance handle. */
-    DWORD reason;		/* Reason this function is being called. */
-    LPVOID reserved;		/* Not used. */
+DllEntryPoint(HINSTANCE hInst, DWORD reason, LPVOID reserved)
 {
     return TRUE;
 }
