--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -70,25 +70,6 @@
 target_include_directories(minhook PRIVATE "src/")
 target_include_directories(minhook PRIVATE "src/hde/")
 
-if(WIN32)
-  set_target_properties(minhook PROPERTIES PREFIX "")
-  if(CMAKE_SIZEOF_VOID_P EQUAL 8)   
-    set_target_properties(minhook PROPERTIES DEBUG_POSTFIX ".x64d")
-    set_target_properties(minhook PROPERTIES RELEASE_POSTFIX ".x64")
-    set_target_properties(minhook PROPERTIES RELWITHDEBINFO_POSTFIX ".x64")
-    set_target_properties(minhook PROPERTIES MINSIZEREL_POSTFIX ".x64")
-  else()
-    set_target_properties(minhook PROPERTIES DEBUG_POSTFIX ".x32d")
-    set_target_properties(minhook PROPERTIES RELEASE_POSTFIX ".x32")
-    set_target_properties(minhook PROPERTIES RELWITHDEBINFO_POSTFIX ".x32")
-    set_target_properties(minhook PROPERTIES MINSIZEREL_POSTFIX ".x32")
-  endif()
-else()
-  set_target_properties(minhook PROPERTIES PREFIX "lib")
-  set_target_properties(minhook PROPERTIES POSTFIX "")
-  set_target_properties(minhook PROPERTIES DEBUG_POSTFIX "d")
-endif()
-
 ################
 # CMAKE CONFIG # 
 ################
